tools/docker/build.sh

```diff
@@ -32,7 +27,7 @@ main() {
         docker_args+=(-v "${PRPLMESH_PLATFORM_BASE_DIR}:${PRPLMESH_PLATFORM_BASE_DIR}")
     fi
 
-    run docker container run "${docker_args[@]}" --rm prplmesh-builder build "$@"
+    run docker container run "${docker_args[@]}" --rm ${DOCKER_REGISTRY}prplmesh-builder build "$@"
```

Comment:
```suggestion
    run docker container run "${docker_args[@]}" --rm "${DOCKER_REGISTRY}"prplmesh-builder build "$@"
```

Reply:
Damn, I was sure I had run shellcheck on it...